pkg/vm/engine/tae/txn/txnimpl/txn.go

```diff
@@ -63,3 +67,97 @@ func (txn *txnImpl) DatabaseNames() (names []string) {
 func (txn *txnImpl) LogTxnEntry(dbId, tableId uint64, entry txnif.TxnEntry, readed []*common.ID) (err error) {
 	return txn.Store.LogTxnEntry(dbId, tableId, entry, readed)
 }
+
+// TODO::ref to MOToVectorTmp in moengine/utils.go
+func MOToTAEVector(v *vector.Vector, nullable bool) containers.Vector {
+	return nil
+}
+
+// TODO::
+func PBToMOVector(v *api.Vector, nullable bool) *vector.Vector {
+	return nil
+}
+
+func MOToTAEBatch(bat *batch.Batch, schema *catalog.Schema) *containers.Batch {
+	//schema := rel.handle.GetMeta().(*catalog.TableEntry).GetSchema()
+	allNullables := schema.AllNullables()
+	taeBatch := containers.NewEmptyBatch()
+	defer taeBatch.Close()
+	for i, vec := range bat.Vecs {
+		v := MOToTAEVector(vec, allNullables[i])
+		taeBatch.AddVector(bat.Attrs[i], v)
+	}
+	return taeBatch
+}
+
+// TODO::
+func PBToMOBatch(bat *api.Batch, schema *catalog.Schema) *batch.Batch {
+	return nil
+}
+
+func (txn *txnImpl) HandleCmd(cmd *api.Entry) (err error) {
+	db, err := txn.GetDatabase(cmd.DatabaseName)
+	if err != nil {
+		return err
+	}
+	//Handle DDL
+	if cmd.DatabaseId == catalog.SystemDBID {
+		switch cmd.TableId {
+		case catalog.SystemTable_DB_ID:
+			if cmd.EntryType == api.Entry_Insert {
+				//TODO:: build schema of database.
```

Comment:
Comment out here are very bad idea -- you silently corrupt a database, right?   Missing insert ...

If you really need to not implement this time and leave it to next PR, panic here.

Reply:
Ok,  i 'll use panic instead.